app/Http/Controllers/Api/LocationsController.php

```diff
@@ -47,11 +63,17 @@ public function index(Request $request)
             'locations.image',
             'locations.ldap_ou',
             'locations.currency',
+            'locations.company_id',
         ])->withCount('assignedAssets as assigned_assets_count')
             ->withCount('assets as assets_count')
             ->withCount('rtd_assets as rtd_assets_count')
             ->withCount('users as users_count');
 
+        // Only scope locations if the setting is enabled
+        if (Setting::getSettings()->scope_locations_fmcs) {
+            $locations = Company::scopeCompanyables($locations);
+        }
```

Comment:
Should this be wrapped in the following like we have in other places?

```
// Only scope the location if the setting is enabled
if (Setting::getSettings()->scope_locations_fmcs) {
    $location->company_id = Company::getIdForCurrentUser($request->input('company_id'));
} else {
    $location->company_id = $request->input('company_id');
}
```

Reply:
This is in line with other scoped controllers and not about the company_id or am I misunderstanding something?